packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart

```diff
@@ -106,6 +117,8 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
       if (_isAtsignOnboarded) {
         await _activateAtsign(atLookUpImpl);
       }
+    } on Exception catch (e) {
+      logger.info('Caught exception: $e');
```

Comment:
change to logger.severe.
Change error message to 'Exception while activating atsign'
add a block for on Error as well